packages/builder/src/components/deploy/clientVersions.json

```diff
@@ -0,0 +1,3 @@
+[
```

Comment:
List of available older client versions that a user can set in dev mode, this is updated automatically by the `getOlderClientVerions` script